homeassistant/components/automation/__init__.py

```diff
@@ -778,6 +787,9 @@ def _trigger_extract_entities(trigger_conf: dict) -> list[str]:
     if trigger_conf[CONF_PLATFORM] in ("state", "numeric_state"):
         return trigger_conf[CONF_ENTITY_ID]
 
+    if trigger_conf[CONF_PLATFORM] == "calendar":
```

Comment:
For a future PR -> let's move this to `trigger.py` for the integration providing the trigger.